features/canvas-2d-context-willreadfrequently.yml

```
@@ -0,0 +1,6 @@
+name: Canvas 2D `willReadFrequently` parameter
```

See comment above about the `alpha` parameter.

---

By name, I'd expect this be "willReadFrequently" all by itself, though if we have to specify the context:

```suggestion
name: `willReadFrequently` for 2D canvases
```

Again though, if this is a hint, do we need to scope this to a specific context?

---

Yes, it is only supported in Canvas 2D